src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -434,6 +434,7 @@ public void testAllXmlExamples() throws Exception {
                 // can't test ant structure, or old and outdated checks
                 assertWithMessage("Xml is invalid, old or has outdated structure")
                         .that(fileName.startsWith("anttask") || fileName.startsWith("releasenotes")
+                                || fileName.startsWith("writingjavadocchecks")
```

Comment:
Please put previous `||` on new line also to ease reading

Reply:
Done.